src/components/MultiHopTrade/helpers.ts

```diff
@@ -85,23 +86,43 @@ export const getNetReceiveAmountCryptoPrecision = ({
   return netReceiveAmountCryptoPrecision.toString()
 }
 
+export const _getHopTotalNetworkFeeFiatPrecision = (
+  tradeQuoteStep: TradeQuote['steps'][number],
+  getFeeAssetRate: (feeAssetId: AssetId) => string,
+): BigNumber => {
+  // TODO(woodenfurniture): handle osmo swapper crazy netowrk fee logic here
```

Comment:
If I was a nitpicker I'd suggest this
```suggestion
  // TODO(woodenfurniture): handle osmo swapper crazy network fee logic here
```